src/main/java/uk/gov/hmcts/reform/bailcaseapi/domain/handlers/presubmit/UploadBailSummaryDocumentHandler.java

```diff
@@ -73,16 +73,16 @@ public PreSubmitCallbackResponse<BailCase> handle(
                     .map(Optional::get)
                     .collect(Collectors.toList());
 
-            Optional<List<IdValue<DocumentWithMetadata>>> maybeExistingBailSummaryDocuments =
-                bailCase.read(UPLOAD_BAIL_SUMMARY_METADATA);
+            Optional<List<IdValue<DocumentWithMetadata>>> maybeExistingHomeOfficeDocuments =
```

Comment:
Please mention in the ticket that this renaming has happened, so that Nilay can test it in one of the scenarios.

Reply:
Done